samples/wifi/sr_coex/prj.conf

```diff
@@ -70,13 +70,15 @@ CONFIG_NET_BUF_TX_COUNT=48
 CONFIG_HEAP_MEM_POOL_SIZE=200000
 CONFIG_NET_BUF_DATA_SIZE=1100
 
+CONFIG_WIFI_CREDENTIALS=y
+CONFIG_WIFI_CREDENTIALS_STATIC=y
 # Below configs need to be modified based on security
-# CONFIG_STA_KEY_MGMT_NONE=y
-# CONFIG_STA_KEY_MGMT_WPA2=y
-# CONFIG_STA_KEY_MGMT_WPA2_256=y
-# CONFIG_STA_KEY_MGMT_WPA3=y
-CONFIG_STA_SAMPLE_SSID="Myssid"
-CONFIG_STA_SAMPLE_PASSWORD="Mypassword"
+# CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_OPEN=y
+# CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_PSK=y
+# CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_PSK_SHA256=y
+# CONFIG_WIFI_CREDENTIALS_STATIC_TYPE_SAE=y
```

Comment:
Do we need all these mentioned under comments? Just set WPA2/WPA3 as default.

Reply:
WPA2 is used as default in library Kconfig, so using the same here and removing other configs.

Reply:
Then we can remove it altogether.